pipeline/gnomad/download_gnomad_data.py

```diff
@@ -0,0 +1,154 @@
+import requests
+import pandas
+import json
+import argparse
+import logging
+
+
+def parse_args():
+        parser = argparse.ArgumentParser(description='Download gnomad data and convert to .tsv format.')
+        parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                            help='Ouput TSV file result.')
+        parser.add_argument('-l', '--logfile', default='/tmp/download_gnomad_data.log')
+        parser.add_argument('-v', '--verbose', action='count', default=False, help='determines logging')
+        options = parser.parse_args()
+        return options
+
+
+def main():
+    options = parse_args()
+    output = options.output
+    logfile = options.logfile
+
+    if options.verbose:
+        logging_level = logging.DEBUG
+    else:
+        logging_level = logging.CRITICAL
+
+    logging.basicConfig(filename=logfile, filemode="w", level=logging_level)
+
+    variants_brca1 = generate_data("BRCA1")
```

Comment:
Gene name should probably be passed in as a program argument, i.e. either a complete list or just one gene. In the latter case, we would call this script for every gene separately.